tests/Rename-DbaDatabase.Tests.ps1

```diff
@@ -12,8 +12,97 @@ Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
         }
     }
 }
-<#
-    Integration test should appear below and are custom to the command you are writing.
-    Read https://github.com/sqlcollaborative/dbatools/blob/development/contributing.md#tests
-    for more guidence.
-#>
\ No newline at end of file
+
+Describe "$commandname Integration Tests" -Tags "IntegrationTests" {
+    BeforeAll {
+        $null = New-DbaDatabase -SqlInstance $script:instance3 -Name 'dbatoolsci_rename1'
+        $date = (Get-Date).ToString('yyyyMMdd')
+    }
+    AfterAll {
+        $database = Get-DbaDatabase -SqlInstance $script:instance3 -UserDbOnly | Where-Object {$_.name -like '*dbatoolsci*'}
```

Comment:
Need to be specific ont he database name in the event someone is using `dbatoolsci`...know it is not likely but just in case only drop the database you created.

Reply:
Okay. I will make this change. I am not fully done with the test, but I will put the actual name in when I get done renaming it.